### ISSUE TITLE
Not reloading | Code - OSS 

### ISSUE THREAD

**Issue by anthonyholstein** [Microsoft/vscode#63177]
Issue Type: <b>Bug</b>

When I reload the window, none of my additions are added to the window.
For example: I modify the page 'vs_code_welcome_page'', I save the file, then I refresh the window on the OSS version then nothing is added. Even if I close and reopen with <i>./scripts/code.sh</i> nothing happens ..

VS Code version: code-oss-dev 1.30.0 (Commit unknown, Date unknown)
OS version: Darwin x64 17.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7 CPU         870  @ 2.93GHz (8 x 2930)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: unavailable_software<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 3, 3|
|Memory (System)|8.00GB (0.77GB free)|
|Process Argv|.|
|Screen Reader|no|
|VM|0%|

</details>Extensions: none

**Comment by roblourens** [comment]
What do you mean, "additions"? Please explain the problem some more

**Comment by anthonyholstein** [comment]
I redid the description of my problem.

**Comment by anthonyholstein** [comment]
I redid the description of my problem. @roblourens

**Comment by roblourens** [comment]
Did you build it after modifying that file?

**Comment by anthonyholstein** [comment]
With which command in the cmd?

**Comment by anthonyholstein** [comment]
With which command in the cmd? @roblourens

**Comment by roblourens** [comment]
That's correct. Can you tell me exactly what you changed and what you expected to see?

**Comment by anthonyholstein** [comment]
Yesterday morning I had modified <p class="subtitle detail">${escape(localize({ key: 'welcomePage.editingEvolved', comment: ['Shown as subtitle on the Welcome page.'] }, "Editing evolved"))}</p> in the vs_code_welcome_page.ts file. I save the file, rebuild, tried to reload the window but nothing worked. It was only 24 hours later, so this morning the window was updated. Why does it take a long time before the changes can appear on the window (VScode - OSS) and it is not instantaneously?
